### PR TITLE
[READY] Add CI check for duplicates in facts

### PR DESCRIPTION
**Pull request by kylerisse**
## Description of PR
Add CI check for duplicates in facts

## Previous Behavior
duplicate data in facts made it into repo which caused pain

## New Behavior
duplicate data in facts should fail CI, thus preventing some pain

## Tests
- locally tested script
- testing flake from this branch

### CODE REVIEW

File: facts/test_duplicates.fish [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env fish
```
Comment (ruebenramirez) [inline facts/test_duplicates.fish:1]: TODO: add fish to dev shell?

File: nix/dev/flake-module.nix [hunk 2]
```diff
@@ -9,6 +9,7 @@
       global = with pkgs; [
         bash
         curl
+        fish
```
Comment (ruebenramirez) [inline nix/dev/flake-module.nix:12]: woohoo!

**Review by ruebenramirez — APPROVED** [review]
🚀

**Review by sarcasticadmin — APPROVED** [review]
Tested a failure and it works as intended

Thanks for catching this

**Pull request merged**